tests/functional/raw-node/package.json

```diff
@@ -8,11 +8,11 @@
     "test"
   ],
   "scripts": {
-    "test-aws": "AWS_ON_AIR=true mocha -t 40000 test/",
-    "test-gcp": "mocha -t 40000 test/GCP/",
-    "test-routes": "mocha -t 40000 test/routes/",
-    "test": "mocha -t 40000 test/",
-    "test-debug": "_mocha -t 40000 test/"
+    "test-aws": "AWS_ON_AIR=true mocha -t 40000 test/ --exit",
+    "test-gcp": "mocha -t 40000 test/GCP/ --exit",
+    "test-routes": "mocha -t 40000 test/routes/ --exit",
+    "test": "mocha -t 40000 test/ --exit",
+    "test-debug": "_mocha -t 40000 test/ --exit"
```

Comment:
this is likely an issue in the tests, which should be fixed so we can remove the flag --> created CLDSRV-611

Reply:
It can also be as simple as http servers we open for the test, but don't properly close (not always the tests themselves)